src/store/modules/network.ts

```diff
@@ -10,14 +10,14 @@ export default {
     networkCurrency: "ETH",
     networkName: "Unsupported Network",
     supportedNetworks: {
-      19: "Songbird"
+      16: "Flare"
```

Comment:
@chnejohnson Please rename it to Coston Testnet. We'll change it to Flare later, after the testing phase is over.